### ISSUE TITLE
bad predictions

### ISSUE THREAD

**Issue by louxy126** [divamgupta/image-segmentation-keras#49]
I got very bad predictions like this（vgg_unt2，epochs=100） 
![Uploading 0016E5_07959.png…]()。
How can i improve it？

**Comment by louxy126** [comment]
![0016e5_07959](https://user-images.githubusercontent.com/37444280/49569015-b1878400-f96d-11e8-8f51-9e2b62e3aebd.png)
![0016e5_07959](https://user-images.githubusercontent.com/37444280/49569034-be0bdc80-f96d-11e8-9009-d090ffa9b86d.png)

**Comment by louxy126** [comment]
It‘s an image from the test floder, I trained the net with 
`parser.add_argument("--epochs", type = int, default = 100 )
parser.add_argument("--batch_size", type = int, default = 4 )
parser.add_argument("--val_batch_size", type = int, default = 4 )
parser.add_argument("--load_weights", type = str , default = "")
parser.add_argument("--input_height", type=int , default = 224  )
parser.add_argument("--input_width", type=int , default = 224 )`
and at the end of predict.py,i resize picture to 640,320 
`	seg_img = cv2.resize(seg_img  , (640 , 320 ))`

**Comment by louxy126** [comment]
prediction with the same image use vgg_segnet
final outcome of train:
512/512 [==============================] - 37s 72ms/step - loss: 0.0897 - acc: 0.9318 - val_loss: 1.0100 - val_acc: 0.8151
![0016e5_07959](https://user-images.githubusercontent.com/37444280/49571720-02e74180-f975-11e8-8eb0-2662c9ee99a8.png)

**Comment by louxy126** [comment]
prediction with the same image use fcn32
final outcome of train:
512/512 [==============================] - 176s 345ms/step - loss: 1.4754 - acc: 0.4158 - val_loss: 1.6122 - val_acc: 0.3327
![0016e5_07959](https://user-images.githubusercontent.com/37444280/49635286-39d35b00-fa3a-11e8-8ff8-f3182cd79446.png)

**Comment by zdd0819** [comment]
hi, i want to ask how do you get the prediction picture? i run the prediction command but i didn't see anything.

many thanks~!
